images/router/haproxy/conf/haproxy-config.template

```diff
@@ -6,7 +6,8 @@
 {{ define "/var/lib/haproxy/conf/haproxy.config" }}
 {{ $workingDir := .WorkingDir }}
 global
-  # maxconn 4096
+  maxconn {{env "ROUTER_MAX_CONNECTIONS" "20000"}}
```

Comment:
Also, while supporting this is ok for now, I don't think this is a value that should be hardcoded in the long run.  The router ultimately needs to make a determination about how many connections to allow based on other factors.  Arbitrary constants that don't work for *every* cluster are a recipe for a break at the worst time.